articles/azure/azs-how-create-scale-set.md

```diff
@@ -0,0 +1,126 @@
+---
+title: How to Create a Scale Set using UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides help for creating a scale set on UKCloud for Microsoft Azure
+services: azure-stack
+author: Shaion O'Niel
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a Scale Set - Portal
+toc_fullpath: Users/How To/azs-how-create-scale-set.md
+toc_mdlink: azs-how-create-scale-set.md
+---
+
+# How to create a Virtual Machine Scale Set using UKCloud Azure Stack portal
+
+## Overview
+
+Virtual Machine scale sets in Azure Stack are a compute resource that can be used to deploy and manage identical virtual machines. All virtual machine instances in a scale set are configured in the same way and do not require any provisioning beforehand.
```

Comment:
Change first sentence to be active rather than passive:
Virtual machine (VM) scale sets in Azure Stack are a compute resource that you can use to deploy and manage identical VMs.